lib/OpenQA/Schema/ResultSet/Comments.pm

```diff
@@ -38,12 +38,7 @@ sub compute_bugref_count {
     my ($self) = @_;
 
     my $comments = $self->search({-not => {job_id => undef}});
-    my %bugrefs;
-    for my $comment ($comments->all) {
-        for my $bug (@{find_bugrefs($comment->text)}) {
-            $bugrefs{$bug} += 1;
-        }
-    }
+    my %bugrefs = map { $_ => 1 } map { @{find_bugrefs($_->text)} } $comments->all;
```

Comment:
Wish we could depend on a newer version of Perl. This is one of the cases where i would really like postderef. Maybe some day.
```perl
my %bugrefs = map { $_ => 1 } map { find_bugrefs($_->text)->@* } $comments->all;
```